pkg/container-utils/cri/cri.go

```diff
@@ -23,11 +23,12 @@ import (
 	"strings"
 	"time"
 
-	runtimeclient "github.com/inspektor-gadget/inspektor-gadget/pkg/container-utils/runtime-client"
 	log "github.com/sirupsen/logrus"
 	"google.golang.org/grpc"
 	"google.golang.org/grpc/credentials/insecure"
-	pb "k8s.io/cri-api/pkg/apis/runtime/v1alpha2"
+	pb "k8s.io/cri-api/pkg/apis/runtime/v1"
```

Comment:
We will need to support both, see:
https://github.com/inspektor-gadget/inspektor-gadget/issues/1507